CMakeLists.txt

```diff
@@ -158,11 +158,15 @@ else()
     set(openPMD_HAVE_ADIOS1 FALSE)
 endif()
 
-# TODO: check!
-#if(openPMD_HAVE_MPI AND openPMD_HAVE_ADIOS AND ADIOS_HAVE_SEQUENTIAL)
-#    message(FATAL_ERROR "Found MPI but requested ADIOS1 is serial."
-#                        "Set openPMD_USE_MPI=OFF to disable MPI.")
-#endif()
+# TODO we could support more combinations than MPI+pADIOS and noMPI+sADIOS
+if(openPMD_HAVE_MPI AND openPMD_HAVE_ADIOS1 AND ADIOS_HAVE_SEQUENTIAL)
+    message(FATAL_ERROR "Found MPI but requested ADIOS1 is serial. "
```

Comment:
are you sure a MPI-parallel ADIOS not not still ship a "sequential" component "as well"?
(also the opposite needs to be checked)

Reply:
The library itself might provide both, but with the used ```FindAdios.cmake```, only mutually exclusive versions get linked. Since ADIOS has a unified interface (with dummy MPI if serial), I doubt you can link both simultaneously.